overview/default.nix

```
@@ -190,12 +190,11 @@ let
 
         </details>
       '';
-      many =
-        examples:
-        optionalString (!empty examples) ''
-          ${heading 2 "examples" "Examples"}
-          ${concatLines (map one examples)}
-        '';
+      many = examples: ''
+        ${heading 2 "examples" "Examples"}
+        <button class="button example"><a class = "heading" href="https://github.com/ngi-nix/ngipkgs/blob/main/CONTRIBUTING.md">Add example</a></button>
+        ${concatLines (map one examples)}
```

```suggestion
        ${concatLines (map one examples)}
        <button class="button example"><a class = "heading" href="https://github.com/ngi-nix/ngipkgs/blob/main/CONTRIBUTING.md">Add an example</a></button>
```

Let's put it below. Once we expand the examples by default, we first of all want users to see the examples, and only secondarily have potential contributors find how to add more.